internal/datastore/crdb/stats.go

```diff
@@ -26,7 +26,7 @@ const (
 
 var (
 	queryReadUniqueID         = psql.Select(colUniqueID).From(tableMetadata)
-	queryRelationshipEstimate = fmt.Sprintf("SELECT COALESCE(SUM(%s), 0) FROM %s AS OF SYSTEM TIME follower_read_timestamp()", colCount, tableCounters)
+	queryRelationshipEstimate = fmt.Sprintf("SELECT array_upper(ARRAY[COALESCE(SUM(%s), 0), 0], 1) FROM %s AS OF SYSTEM TIME follower_read_timestamp()", colCount, tableCounters)
```

Comment:
since we know why these can go negative, maybe just read them into a signed int and `min` check them on the Go side?